ngraph/python/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ endif()
 set(PYTHON_VERSION python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR})
 message("Python version=${PYTHON_VERSION}")
 
+set(LIBRARY_OUTPUT_DIRECTORY_BIN ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
 if(OpenVINO_MAIN_SOURCE_DIR)
```

Comment:
Please take a look to the body of this if. Here we also change `CMAKE_..._OUTPUT_DIRECTORY`. Can we use the approach with `target_property`?

Reply:
As discussed, we've decided to keep it as is and don't rework existing cmake as both approaches work well here